schemas/Manual.items.schema.json

```diff
@@ -26,76 +26,78 @@
                     "type": "string"
                 },
-                "progression": {
-                    "description": "(Optional) Is this item needed to unlock locations? Defaults to false if not included.",
-                    "default": true,
-                    "type": "boolean"
-                },
-                "count": {
-                    "description": "(Optional) Number of copy of this item in the pool.",
-                    "type": "integer",
-                    "default": 1
-                },
                 "category": {
                     "description": "(Optional) The category(ies) this item fit in",
                     "type": "array",
                     "items": {
-                        "$ref": "#/definitions/Category"
+                        "type": "string"
                     },
                     "minItems": 1,
                     "uniqueItems": true
                 },
-                "useful": {
-                    "description": "(Optional) Is this item usefull to have? Used for items that are not progression but you still want the rando to really use.",
-                    "type": "boolean"
+                "count": {
+                    "description": "(Optional) Number of copy of this item in the pool.",
+                    "type": "integer",
+                    "default": 1
+                },
+                "value": {
+                    "description": "(Optional) A dictionary of values this item has in the format {\"name\": int,\"otherName\": int} \nUsed with the {ItemValue(Name: int)} in location requires \neg. \"value\": {\"coins\":10} mean this item is worth 10 coins",
+                    "type": "object",
+                    "patternProperties": {
+                        "^.+$": {
+                            "anyOf": [
+                                {
+                                    "type": "integer",
+                                    "description": "A value that this item has must be 'name':integer \neg. \"coins\": 10"
+                                }
+                            ]
+                        }
+                    }
+                },
+                "progression": {
+                    "description": "(Optional) Is this item needed to unlock locations? Defaults to false if not included.",
+                    "type": "boolean",
+                    "default": false
                 },
                 "progression_skip_balancing": {
                     "description": "(Optional) Should this item not get included in progression balance swaps. For more info check the discord",
                     "type": "boolean",
-                    "default": true
+                    "default": false
+                },
+                "useful": {
+                    "description": "(Optional) Is this item usefull to have? Used for items that are not progression but you still want the rando to really use.",
```

Comment:
check spelling on "useful"

Reply:
Fixed that as well. Reworked all of the item classification descriptions to be more uniform and more concisely describe themselves.